chai-immutable.js

```diff
@@ -437,10 +438,15 @@ module.exports = function (chai, utils) {
    */
 
   assert.equal = function (actual, expected) {
+    /*
+     * It seems like we shouldn't actually need this check, however,
+     * `assert.equal` actually behaves differently than its BDD counterpart!
+     * Namely, the BDD version is strict while the "assert" one isn't.
+     */
```

Comment:
Good point to add that comment, I was actually just thinking "wait, why do I just and only call `equal` in the equal assertion?!".